system/Config/Services.php

```diff
@@ -234,9 +234,9 @@ public static function exceptions(
             return static::getSharedInstance('exceptions', $config, $request, $response);
         }
 
-        $config   = $config ?? config('Exceptions');
-        $request  = $request ?? AppServices::request();
-        $response = $response ?? AppServices::response();
+        $config ??= config('Exceptions');
+        $request ??= AppServices::request();
+        $response ??= AppServices::response();
```

Comment:
Probably a style question for @paulbalandan - are these intentionally exempt from alignment?

Reply:
By default, our binary operators alignment is set to `single_space` to avoid surprises. We just add some exceptions. This could be added to that list.

Reply:
I think this would make sense to include. I guess the question is how these operators interact with each other, e.g.:
```php
$fruit = 'banana';
$vegetable ??= 'celery';

// versus
$fruit       = 'banana';
$vegetable ??= 'celery';
```

Reply:
PHP-CS-Fixer is smart enough to distinguish the two, as it relies on tokens. `=` is `=` while `??=` is `T_COALESCE_EQUAL`